release.toml

```diff
@@ -1,6 +1,7 @@
 enable-all-features = true
 pre-release-hook = [
   "../../.github/gen_changelog.sh \"$CRATE_ROOT\" \"$NEW_VERSION\"",
+  "git add \"*CHANGELOG.md\"",
```

Comment:
I don't think this will work

Reply:
Weird that it's an array but I trust you https://github.com/iotaledger/iota-rust-sdk/pull/413/commits/6919672b597e7a2509ecab4ad10517bb63eda6a2

Reply:
https://github.com/iotaledger/iota-rust-sdk/pull/413/commits/f8841606218ff07cd18da6585531578f98316ca9